doc/nrf/app_dev/device_guides/nrf91/nrf91_board_controllers.rst

```diff
@@ -26,7 +26,7 @@ The nRF5340 IMCU comes preprogrammed with J-Link SEGGER OB and board controller
 If you want to change the default configuration of the DK, you can use the `Board Configurator app`_ in `nRF Connect for Desktop`_ .
 
 To update board controller firmware on the nRF9161 DK, download the `nRF9161 DK board controller firmware`_ from the nRF9161 DK downloads page.
-To program the HEX file, use `nRF Util`_.
+To program the HEX file, use the `nRF Util`_ tool.
```

Comment:
Do not use `the` and `tool` with this link. This is not the case throughout the docs.